api/lib/domain/usecases/get-certification-point-of-contact.js

```diff
@@ -20,3 +30,14 @@ const getCertificationPointOfContact = async function ({
 };
 
 export { getCertificationPointOfContact };
+
+/**
+ * @param {Object} params
+ * @param {CenterRepository} params.centerRepository
+ * @returns {Array<number>}
```

Comment:
On renvoi des centres si je comprends bien? ()
```suggestion
 * @returns {Array<Center>}
```

Reply:
Done